package.json

```diff
@@ -1,4 +1,10 @@
 {
+  "scripts": {
+    "pretest": "rm -rf .jest-cache",
+    "test": "jest --no-cache",
+    "test:watch": "jest --watch --no-cache",
+    "test:ci": "jest --ci --runInBand --no-cache --coverage"
```

Comment:
why are we disabling the cache for the tests. It does make jest "at least two times slower" https://jestjs.io/docs/en/cli#--cache

Reply:
This is so the CI would run in a clean slate all the time. But might be worth using `--clearCache` instead

Reply:
Would CI ever have previous cache?  TBH I don't think I've ever come across issues with caching issues in CI with jest tests.

Reply:
Yup happy to remove this

Reply:
removed this change from this pr